src/RequestBuilder/SortingRequestBuilder.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\Sorting;
+use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\Model\Request;
+
+class SortingRequestBuilder extends AbstractRequestBuilder
+{
+    protected const ENDPOINT_PATH = '/sorting';
+
+    /** @var Interaction */
+    private $interactionCommand = null;
```

Comment:
Null is default, no need to explicitly define it as a value. Maybe rather in the phpdoc, to say its "legal" value.